Bottom: bff779db354e313c0743548a5ef661a763005024
Top:    c5c0a8d77b30dce5118ede52a82222293855e48c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-04-14 10:11:15 -0700

Refresh of rs-iwarp

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 4caa93a..f529a3c 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1548,6 +1548,9 @@ static int rs_post_msg(struct rsocket *rs, uint32_t msg)
 		wr.send_flags = 0;
 		wr.imm_data = htonl(msg);
 	} else {
+		sge.addr = (uintptr_t) &msg;
+		sge.lkey = 0;
+		sge.length = sizeof msg;
 		wr.sg_list = &sge;
 		wr.num_sge = 1;
 		wr.opcode = IBV_WR_SEND;
